### PR TITLE
Docker builds failing #72

### PR DESCRIPTION
**Pull request by fredrikhgrelland**
## Checklist (after created PR)
- [ ] Added reviewers
- [ ] Added assignee(s)
- [ ] Added label(s)
- [ ] Added to project
- [ ] Added to milestone

### CODE REVIEW

File: conf/nomad/hive.hcl [hunk 1]
```diff
@@ -59,16 +59,34 @@ job "${service_name}" {
...
+          "-w",
+          "HTTP code: %%{http_code}",
+          "127.0.0.1:${minio_local_bind_port}/minio/health/ready"
+        ]
         interval = "15s"
         timeout  = "5s"
-        address_mode = "driver"
       }
-
+      */
```
Comment (pdmthorsrud) [inline conf/nomad/hive.hcl:78]: Maybe I missed something while being a little in and out of the call, but why is this commented out?

Comment (claesgill) [reply conf/nomad/hive.hcl:78]: The test failed again on friday, so I guess Fredrik commented it out to isolate the error.

**Review by claesgill — APPROVED** [review]

File: conf/nomad/hive.hcl (hunk 1 above)
Comment (pdmthorsrud) [reply conf/nomad/hive.hcl:78]: If that's what it is, I'm cool with it too

**Review by pdmthorsrud — APPROVED** [review]
LGTM

**Pull request merged**